py/openage/codegen/codegen.py

```diff
@@ -6,7 +6,7 @@
 
 from openage.log import dbg
 
-contenttemplate = """\
+contenttemplate = """
```

Comment:
According to @TheJJ the backslash is not only not necessary, it actually shows up in the generated docs. Therefore I removed them all.